source/windows/system_info.c

```diff
@@ -299,4 +299,19 @@ void aws_backtrace_log() {
     AWS_LOGF_TRACE(
         AWS_LS_COMMON_GENERAL, "aws_backtrace_log: backtrace requested, but logging is unsupported on this platform");
 }
+
+int aws_get_memory_usage(int64_t *output) {
+    // Windows is currently not supported.
+    *output = 0;
+    aws_raise_error(AWS_ERROR_PLATFORM_NOT_SUPPORTED);
```

Comment:
surely windows offers something like this ...

googling found: [GlobalMemoryStatusEx()](https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-globalmemorystatusex)

I'm hesitant to add simple functions to aws-c-common that we didn't implement for common platforms.